steps/zip-directory-and-export-its-path/v1.0.0/step.yml

```diff
@@ -0,0 +1,55 @@
+title: Zip Directory and Export its Path
+summary: |
+  This step will zip a directory and then export the path as an environment variable
+description: |
+  This step will zip a directory and then export the path as an environment variable
+website: https://github.com/danielmcconville/bitrise-step-zip-directory-and-export-its-path
+source_code_url: https://github.com/danielmcconville/bitrise-step-zip-directory-and-export-its-path
+support_url: https://github.com/danielmcconville/bitrise-step-zip-directory-and-export-its-path/issues
+published_at: 2018-11-16T13:04:35.735930269+10:30
+source:
+  git: https://github.com/danielmcconville/bitrise-step-zip-directory-and-export-its-path.git
+  commit: 7533b3a183c751d7e81f4433f932defb287de21d
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: zip
+  apt_get:
+  - name: zip
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- directory_to_zip: $BITRISE_DIRECTORY_TO_ZIP
+  opts:
+    description: |
+      Define the path to the directory you want zipped here.
+    is_expand: true
+    is_required: true
+    summary: Path to the directory to zip.
+    title: Path to the directory to zip.
+- include_directory: true
+  opts:
+    description: "Set to true to include the parent directory in the zip file, false
+      if you \njust want the files in the directory.\n"
+    is_expand: false
+    is_required: true
+    summary: If set to true, the zip file will include the parent directory.
+    title: Include the directory in the zip
+    value_options:
+    - true
```

Comment:
Please change the value options from bool to string:
```
value_options:
    - "true"
    - "false"
```

Reply:
Will do, thanks.